openqabot/types/incidents.py

```diff
@@ -236,6 +236,15 @@ def __call__(
 
                     # add custom vars to job settings
                     if "params_expand" in data:
+                        if any(
```

Comment:
the indendation level seems to become excessive here. Could you please try to extract this and the neighboring code parts into a function so that the level of indendation can be reduced here?

Reply:
`black` does not blame me about it

Reply:
@okurz can you unblock this merge? There already 2 approvals and @mpagot will address your suggestion in an upcoming PR (we're not even close to being done with this task)

Reply:
> But it means a big change, big just from the point of view of how many lines of code are moved around.
> I propose to do it in a next PR.
> How it sound?

Sorry, but this should be done as part of this PR. Make it a separate commit, sure, but still part of this PR. Unfortunately I know how often it happens that after a change someone has no time to work on promised changes anymore and then the codebase deteriorates and invites for more mistakes by others. Maybe it wouldn't happen here but that's not an unusual experience. So please look into this before merging.

Reply:
Nevermind. As baierjan stated "It's OK I guess." :)